packages/api/src/index.js

```diff
@@ -104,6 +105,9 @@ r.add('get', '/internal/tokens', withMode(tokensList, RO), [postCors])
 r.add('post', '/internal/tokens', withMode(tokensCreate, RW), [postCors])
 r.add('delete', '/internal/tokens', withMode(tokensDelete, RW), [postCors])
 
+// Blog
+r.add('post', '/blogSubscribe', withMode(blogSubscribe, RW), [postCors])
```

Comment:
Please can we put this under the /internal namespace so people know it's not a public API?